src/engine/Engine.cpp

```diff
@@ -3555,7 +3555,14 @@ bool Engine::certifyUNSATCertificate()
         groundLowerBounds[i] = _groundBoundManager.getLowerBound( i );
     }
 
-    Checker unsatCertificateChecker( _UNSATCertificate, _tableau->getM(),  _tableau->getSparseA(),
+    if ( GlobalConfiguration::WRITE_JSON_PROOF )
+    {
+        File file ( "proof.json" );
```

Comment:
File file ( --> File file(
Also, make the name a configurable parameter, or at least a constant somewhere at the top of the file